donuts.html

```diff
@@ -0,0 +1,50 @@
+<script>
+  function Store(name, hrsOp, hrlyFt, percentEnt, qtyPer) {
+      this.name = name;
+      this.hrsOp = hrsOp;
+      this.hrlyFt = hrlyFt;
+      this.percentEnt = percentEnt;
+      this.qtyPer = qtyPer;
+      this.hrlyPurch = [];
+      this.total = 0;
+      /*Borrowed code below from http://stackoverflow.com/questions/1527803/generating-random-numbers-in-javascript-in-a-specific-range.  Calculates how many customers enter based on random traffic generated from function and then multiplies by average donuts purchased per customer*/
+      this.ranTraffic = function getRandomInt(min, max) {
+        var ranTraf = Math.floor(Math.random() * (max - min + 1)) + min;
+        return ranTraf;
```

Comment:
This return statement means that the function is ignoring any code below it in the function. (Once you return, the function is done.) So, you're actually console.logging the number selected from random traffic per hour, not the number of donuts purchased.